runtime/syntax/nftables.yaml

```diff
@@ -6,14 +6,23 @@ detect:
     
 rules:
     - type: "\\b(chain|counter|map|rule|ruleset|set|table)\\b"
-    - type: "\\b(ether|icmp|icmpv6|icmpx|inet|ip|ip6|ipv4|ipv6|tcp|udp)\\b"
+    - type: "\\b(ether|inet|i(cm)?p(x|(v?(4|6))?)|tcp|udp)\\b"
     - special: "\\b(elements|hook|policy|priority|type)\\b"
     - identifier: "\\b(ct|iif|iifname|meta|oif|oifname|th)\\b"
     - statement: "\\b(accept|drop|goto|jump|log|masquerade|reject)\\b"
     - preproc: "\\b(add|define|flush|include|delete)\\b"
-    - symbol: "[-=/:;,@]"
+    - symbol: "([-=/:;,@]|[!=])"
     - symbol.operator: "[<>.&|^!]|\\b(and|ge|gt|le|lt|or|xor)\\b"
     - constant.string: '([\"]{1})(.*)([\"]{1})'
+    - constant.string: 
+         start: "(['\"])" 
+         end: "(['\"])" 
+      # Integer Constants
+    - constant.number: "(\\b([1-9][0-9]*|0[0-7]*|0[Xx][0-9A-Fa-f]+|0[Bb][01]+)([Uu][Ll]?[Ll]?|[Ll][Ll]?[Uu]?)?\\b)" 
+      # Decimal Floating Constants 
+    - constant.number: "(\\b(([0-9]*[.][0-9]+|[0-9]+[.][0-9]*)([Ee][+-]?[0-9]+)?|[0-9]+[Ee][+-]?[0-9]+)[FfLl]?\\b)" 
```

Comment:
I'd say only dec and hex is allowed, but no float. But here I'm a bit unsure...need to check at work with the bandwidth limitations.

Reply:
That was also a suggestion. Maybe you typed the wrong line numbers. Because i added just the comment from c language.

Reply:
> I'd say only dec and hex is allowed, but no float. But here I'm a bit unsure...need to check at work with the bandwidth limitations.

I checked it and indeed float numbers shouldn't be added so far.

Reply:
Decimal Floating Constants will be removed with the next commit.